### DISCUSSION TITLE
How to turn windows back into panels?

### DISCUSSION THREAD

**Discussion by TheJanzap** [puddletag/puddletag#833] [User Support] [answered]
While testing out puddletag, I accidentally turned a panel into a floating window and am now unsure how I can turn it back into a side panel. How can I do this?

**Comment by josevidalrt** [comment]
> While testing out puddletag, I accidentally turned a panel into a floating window and am now unsure how I can turn it back into a side panel. How can I do this?

As far as I know, simply dragging it to the edge turns it into a panel.

**Reply by TheJanzap** [reply]
The edge of the main window? That doesn't seem to work. Is this a problem with the Flatpak package?

**Reply by bombseb** [reply]
Hello,

I have the same problem. I unpacked the "Tag panel" window and I can't unpack it again.
I'm on KDE, and I've installed the normal version of Puddletag (not the flatpak version)

**Reply by mikae1** [reply]
> have the same problem. I unpacked the "Tag panel" window and I can't unpack it again.

Same, right here right now. Can't dock after undocking the panel. 2.5.0 Flatpak on KDE Plasma v6.4.4.

**Comment by corubba** [comment, marked as answer]
In general, you can usually just drag them back in as mentioned by @josevidalrt. Depending on your desktop environment/window manager, you may also have little buttons in the top right of the widgets to pop them out of the main window and also to pop them back in. The last resort is to delete `~/.config/puddletag/qt.conf`, which will reset the whole window layout and all widgets.

**Comment by NeuroDawg** [comment]
I'm bringing up this question, as I am unable to drag a window and turn it back into a panel. 

I've installed puddletag on Ubuntu 24.04. I accidentally hit the button to turn a panel to a window. There is not button to turn it back to a panel; all I have is minimize, maximize, and close buttons. When I try to drag to the edge of the puddletag window, the window just goes past and nothing happens. 

I can move panels from side to side, or reorder on the same side, and they stay panels. I just can't get them back to a panel if I've accidentally made them windows.

Any suggestions would be appreciated.

**Reply by corubba** [reply]
Depending on your specific environment, #762 may be relevant here.
